themes/jaeger-docs/layouts/partials/home/articles.html

```diff
@@ -1,9 +1,54 @@
+{{- $maxBlogPosts := .Site.Params.maxBlogPosts }}
+{{- $json         := getJSON "https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/jaegertracing" }}
+{{- $posts        := first $maxBlogPosts $json.items }}
+{{- $mostRecent   := index $json.items 0 }}
+{{- $newsletters  := (slice) }}
+{{- range $posts }}
+{{- if in .categories "newsletter" }}
+{{- $newsletters = $newsletters | append . }}
+{{- end }}
```

Comment:
looks like these 6 lines are not needed, since the newsletter is determined elsewhere

Reply:
Ah, true. I need to make this more DRY anyway. Hold off on merging.

Reply:
@yurishkuro Okay, good to go.